02_bubble/telkar/bubble.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
```

Comment:
Make a copy of `s` first, otherwise you will modify the input.
Write a test ensuring that the input does not get modified.